include/pmacc/particles/tasks/TaskParticlesSend.hpp

```diff
@@ -22,7 +22,8 @@
 #pragma once
 
 #include "pmacc/Environment.hpp"
-#include "pmacc/eventSystem/EventSystem.hpp"
+#include "pmacc/eventSystem/Manager.hpp"
+#include "pmacc/eventSystem/tasks/MPITask.hpp"
```

Comment:
@psychocoderHPC  What was added in the code that now needs `MPITask.hpp`

Reply:
before it was transitively included via `EventSystem.hpp`